atom/browser/web_contents_preferences.cc

```diff
@@ -194,6 +200,21 @@ void WebContentsPreferences::AppendExtraCommandLineSwitches(
     command_line->AppendSwitch(cc::switches::kEnableBeginFrameScheduling);
 }
 
+bool WebContentsPreferences::IsSandboxed(content::WebContents* web_contents) {
+  WebContentsPreferences* self;
+  if (!web_contents)
+    return false;
+
+  self = FromWebContents(web_contents);
+  if (!self)
+    return false;
+
+  base::DictionaryValue& web_preferences = self->web_preferences_;
+  bool sandboxed = false;
+  web_preferences.GetBoolean("sandboxed", &sandboxed);
```

Comment:
> Maybe just sandbox? The ed suffix seems redundant.

:+1: 

> And can you also add this option to docs? 

Sure, I was planning to update the docs after reviews.

> We should also add a new page on which features are allowed in the sandboxed page and how to use it.

As I mentioned in my previous comments, I think it is possible to implement most feature available for normal node.js enabled renderers(even node.js API, which can be proxied through ipcRenderer), but I'd rather do it in future PRs to limit the size of this PR.

If its ok with you, I'd rather document `sandbox` option limitations in a future PR, after I've refactored enough to put as many features in sandboxed renderers as possible.